VideoLocker/src/androidTest/java/org/edx/mobile/test/feature/config/E2EConfig.java

```diff
@@ -0,0 +1,6 @@
+package org.edx.mobile.test.feature.config;
+
+public class E2EConfig {
+    public static String EMAIL_TEMPLATE = "test-{unique_id}@example.com";
```

Comment:
And the password can also be a uuid. Since we run those against prod it's nice to use something with some randomness in the password